setup.py

```diff
@@ -23,7 +23,9 @@ def open_file(fname):
     author='Adam Cohen',
     author_email='adam@seatgeek.com',
     packages=['thefuzz'],
-    extras_require={'speedup': ['python-levenshtein>=0.12']},
+    # keep for backwards compatibility of projects depending on `thefuzz[speedup]`
+    extras_require={'speedup': []},
+    install_requires= ['rapidfuzz>=2.1.2'],
     url='https://github.com/seatgeek/thefuzz',
     license="GPLv2",
```

Comment:
from your PR description, is it possible to already update the licence in this PR?

Reply:
One remaining problem is that large parts of the `process.py` got written after the license was changed. All other files got rewritten (except for docs) in this PR and should not be a problem. Still this would require us to ask all contributors to this file since 2011.

Reply:
from the [fuzz.py blame](https://github.com/seatgeek/thefuzz/blame/193a953c90c91dcbdc1e17adcdad439776cd56f6/thefuzz/fuzz.py) (after this PR) it looks like only you and @hugovk are left as contributors, and the [process.py blame](https://github.com/seatgeek/thefuzz/blame/193a953c90c91dcbdc1e17adcdad439776cd56f6/thefuzz/process.py) adds @kianmeng and @gwerbin.

are you three OK with changing to a more open licence?

Reply:
Probably, which licence did you have in mind?

Reply:
When moving to this repository a lot of the contributors got lost, but they are obviously still license holders. So you would need to check contributors in https://github.com/seatgeek/fuzzywuzzy.
I think the best way to go about this is:
1) merge this PR without a license change
2) create a new PR with the license change and try to ping all of the remaining contributors (I can look into who remains). Try to get as many of them to agree with the license change as possible in the PR. 
  - for people who do not agree -> replace their code
  - for unreachable people -> try to reach them by different means of replace their code
 The implementations in process.py are all pretty simple, so as long as we get the agreement of a large chunk of these contributors this would not be an issue.
 Obviously in regards to the second point the first question is whether SeatGeek would be willing to update the license in case all relevant contributors agree

Reply:
> Probably, which licence did you have in mind?

This is about moving back to the original MIT license

Reply:
MIT is fine by me 👍